prog/dftb+/include/common.fypp

```diff
@@ -40,9 +40,12 @@
 #! Check a condition if WITH_ASSERT is True and call assertError if condition is False.
 #:def ASSERT(cond)
   #:if WITH_ASSERT
-  if (.not. (${cond}$)) then
-    call assertError("${_FILE_}$", ${_LINE_}$)
-  end if
+  block
+    use dftbp_assert, only : assertError
+    if (.not. (${cond}$)) then
+      call assertError("${_FILE_}$", ${_LINE_}$)
```

Comment:
While are on it, can we pass the actual condition tested as string to the assertion error as well?

Reply:
Yes, that would make sense, indeed. I'll add it in a separate PR.